public/app/plugins/panel/annolist/AnnotationListItemTags.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC, MouseEvent } from 'react';
+
+import { GrafanaTheme } from '@grafana/data';
+import { css } from 'emotion';
+import { useStyles } from '@grafana/ui';
+
+import { TagBadge } from '../../../core/components/TagFilter/TagBadge';
+
+interface Props {
+  tags?: string[];
+  remove?: boolean;
+  onClick: (tag: string, remove?: boolean) => void;
+}
+
+export const AnnotationListItemTags: FC<Props> = ({ tags, remove, onClick }) => {
+  const styles = useStyles(getStyles);
+  const onTagClicked = (e: MouseEvent, tag: string) => {
```

Comment:
useCallback maybe? Wouldn't matter from perf perspective, but I believe it's a good practice to memoize handlers

Reply:
Hmm, I've been trying to cut down on memoized handlers. But I can add it, thanks for pointing it out.